### PR TITLE
OPCT-4: User Documentation clarification about the entire process

### PR DESCRIPTION
**Pull request by mtulio**
Clarifying the process to submit the results according to the VCSP Program requirements.

This is an improvement based on the review process.

Expected to be delivered on v0.3 https://issues.redhat.com/browse/OPCT-4

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -169,10 +173,9 @@ How to submit OpenShift Certification Test results:
 
 Required files to attach to a NEW support case:
 
-- Attach the Installation Document used describing how your installing the Cluster
-- Download, review and attach the [`user-installation-checklist.md`](./user-installation-checklist.md)
-- Attach the `retrieved-archive.tar.gz` result file to the case.
-- Attach the `must-gather.tar.gz` file to the case.
```
Comment (mtulio) [inline docs/user.md:175]: v0.2+ already collects automatically the must-gather.

**Comment by bostrt** [comment]
I think we are awaiting some more feedback from Ju so I'll just give the lgtm for now:
/lgtm

File: docs/user.md [hunk 2]
```diff
@@ -28,15 +28,21 @@ Table Of Contents:
 
 ## Process <a name="process"></a>
 
-More detail on each step can be found in the sections further below.
+Overview of the process to apply the results to the VCSP Program:
```
Comment (mtulio) [inline docs/user.md:31]: We need to wait for the Program manager to check if we need to mention the program here.
https://coreos.slack.com/archives/C02DSMAP4LA/p1670882046186029

**Comment by mtulio** [comment]
Removing any reference to the program.

**Comment by mtulio** [comment]
ptal @rvanderp3 
/assign @rvanderp3

File: docs/user.md [hunk 3]
```diff
@@ -32,15 +32,24 @@ Table Of Contents:
 
 ## Process Overview <a name="process"></a>
 
-More detail on each step can be found in the sections further below.
+This section describes the steps of the process when submiting the results to Red Hat Partner,
+if the goal is not sharing the results to Red Hat, you can go to the next section.
```
Comment (rvanderp3) [inline docs/user.md:36]:
```suggestion
This section describes the steps of the process when submiting the results to Red Hat Partner.
If the goal is not sharing the results to Red Hat, you can go to the next section.
```

File: docs/user.md [hunk 4]
```diff
@@ -32,15 +32,24 @@ Table Of Contents:
...
+0. Install an OpenShift cluster on **the version desired** to be validated
+1. Prepare the OpenShift cluster to run the validated environment
+2. Download and install the OPCT
+3. Run the OPCT
 4. Monitor tests 
 5. Gather results
-6. Share certification results and [must gather](https://docs.openshift.com/container-platform/latest/support/gathering-cluster-data.html) with Red Hat
+6. Share the results archive with Red Hat Partner support case
+
+You may be asked by Support Engineers to repeat the process a few times depending on the results.
```
Comment (rvanderp3) [inline docs/user.md:48]:
```suggestion
It's not uncommon for specific tests to occasionally fail.  As a result, you may be asked by Support Engineers to repeat the process a few times depending on the results.
```

File: docs/user.md [hunk 5]
```diff
@@ -32,15 +32,24 @@ Table Of Contents:
...
 
+Finally, you will be asked to manually upgrade the cluster to the next Y-release.
```
Comment (rvanderp3) [inline docs/user.md:50]: Y-release may be an uncommon nomenclature outside of Red Hat.  You might consider 'minor release'.

**Review by rvanderp3 — COMMENTED** [review]
added a few minor comments.  otherwise it looks good!

File: docs/user.md (hunk 5 above)
Comment (mtulio) [inline docs/user.md:50]: @rvanderp3 wdyt?
```suggestion
Finally, you will be asked to manually upgrade the cluster to the next minor release.
```

**Review by mtulio — COMMENTED** [review]
addressing the review

File: docs/user.md (hunk 5 above)
Comment (mtulio) [reply docs/user.md]: Thanks! Ptal?

**Pull request merged**